.github/workflows/make-release.yml

```diff
@@ -127,3 +127,80 @@ jobs:
         with:
           body_path: ${{github.workspace}}/release.md
           prerelease: ${{steps.extract_prerelease.outputs.prerelease}}
+  createhomebrew:
+    runs-on: ubuntu-latest
+    needs: [makerelease]
+    name: createrelease
+    steps:
+      - uses: actions/checkout@master
+      - name: 'Authenticate to Google Cloud'
+        uses: 'google-github-actions/auth@v0.8.0'
+        with:
+          credentials_json: '${{ secrets.GOOGLE_CLOUD_DL_SA }}'
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@v0.6.0
+        with:
+          project_id: kittycadapi
+      - name: Install deps
+        shell: bash
+        run: |
+          ./.github/workflows/cross-deps.sh
+      - name: bundle binaries and make formula
+        shell: bash
+        run: |
+          mkdir homebrew
```

Comment:
fwiw I had tried to put this script into a `.sh` file but would always fail on `input_names=(`unexpected token `(`. Tried a few thing to get it working but couldn't figure it out.